docs/onboarding/Configuration-Properties.md

```diff
@@ -0,0 +1,141 @@
+# Configuration Properties
+
+The Onboarding Server uses the following public configuration properties:
+
+## Database Configuration
+
+| Property | Default | Note |
```

Comment:
Do we have any agreement about table formatting in markdown? To format or not to format? I would rather format.

Reply:
I use the same formatting as in PowerAuth server. If we want to change it, let's discuss it.